layers/+lang/lean/config.el

```diff
@@ -0,0 +1,16 @@
+;;; config.el --- Lean Layer config File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(configuration-layer/declare-layers '(auto-completion lua syntax-checking))
+
```

Comment:
The main package `lean-mode` calls `(require 'company-mode)` and `(require flycheck)` and so it doesn't work if those packages aren't installed. What I could do is to only install `lean-mode` only if those layers are installed, but I thought it would be easier for the use if I just install the packages for her.